src/test/unit/EigenPodUnit.t.sol

```diff
@@ -532,4 +541,11 @@ contract EigenPodUnitTests is Test, ProofParsing {
         eigenPodManager.createPod();
         cheats.stopPrank();
     }
+
+    function _getLatestDelayedWithdrawalAmount(address recipient) internal view returns (uint256) {
```

Comment:
Can you move this helper function in EigenPod.t.sol since no tests in this file use `_getLatestDelayedWithdrawalAmount`